tests/test_cpmorphology.py

```diff
@@ -1165,7 +1169,7 @@ def test_01_02_circle(self):
         labels = np.zeros((1001,1001),int)
         y,x = np.mgrid[-500:501,-500:501]
         labels[x*x+y*y<=250000] = 1
-        extents = morph.calculate_extents(labels,[1])
+        extents = morph.calculate_extents(labels,[1])[0]
```

Comment:
What's the reason for this change?

Reply:
@AetherUnbound `calculate_extents` returns an array. I'm not sure whether there was a change in the behaviour of NumPy or unittest (Py3) since this test was written, I presume it's NumPy, but currently you can't use `unittest.TestCase.assertAlmostEqual` for *approximate* equality of an array of shape (1,) to a number:

```python
In [14]: c = unittest.TestCase()

In [15]: result = np.pi / 4 + np.random.random(1) / 400 - (1/800)

In [16]: result
Out[16]: array([ 0.7859997])

In [17]: np.pi / 4
Out[17]: 0.7853981633974483

In [18]: c.assertAlmostEqual(result, np.pi / 4, 2)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-21-cec72c7712b5> in <module>()
----> 1 c.assertAlmostEqual(result, np.pi / 4, 2)

~/conda/envs/36/lib/python3.6/unittest/case.py in assertAlmostEqual(self, first, second, places, msg, delta)
    860                 places = 7
    861
--> 862             if round(abs(second-first), places) == 0:
    863                 return
    864

TypeError: type numpy.ndarray doesn't define __round__ method
```

Grabbing that element works:

```python
In [19]: c.assertAlmostEqual(result[0], np.pi / 4, 2)
```

(no output, as expected)

Amusingly/confusingly, `assertAlmostEqual` checks for exact equality first, which is why other similar test cases pass:

```python
In [20]: c.assertAlmostEqual(np.array([1]), 1, 2)
```

I think NumPy might be considering removing the implicit conversion of `np.array([True])` to `True`, so it would be good to fix *all* similar tests, but I consider that beyond the scope of this PR. ;)

Reply:
Makes sense! Thanks for the explanation. 👍